.bsp/sbt.json

```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.6.2","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/jdk-13.0.1.jdk/Contents/Home/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/anu/Library/Application Support/JetBrains/IdeaIC2020.1/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp","--sbt-launch-jar=/Users/anu/Library/Application%20Support/JetBrains/IdeaIC2020.1/plugins/Scala/launcher/sbt-launch.jar"]}
```

Comment:
Please remove this file from the commit

Reply:
ok

Reply:
it's probably better to delete the file and add .bsp to the .gitignore file, saves this coming up repeatedly